web/templates/home/index.html

```diff
@@ -2,4 +2,18 @@
 
 <h1>Welcome to iTrust: Zakat management application</h1>
 <h3>Under Construction ...</h3>
-{% endblock %}
+
+{% for case in cases %}
+<div class="card">
+    <div class="card-body">
```

Comment:
Can we have different sections here ?  Open /  Pending Verification /  Pending Voting etc ?

Reply:
To have different sections we need to list all the cases irrespective of case state right.

Reply:
put different state in different cards. and keep in same page.  Or you can keep in different tabs in same card .